### PR TITLE
Correct Bernardi-Raugel element

### PR DESCRIPTION
**Pull request by mscroggs**

### CODE REVIEW

File: symfem/elements/bernardi_raugel.py [hunk 1]
```diff
@@ -137,6 +130,6 @@ def polynomial_superdegree(self) -> typing.Optional[int]:
     references = ["triangle", "tetrahedron"]
     min_order = 1
     max_order = {"triangle": 1, "tetrahedron": 2}
-    continuity = "H(div)"
+    continuity = "L2"
```
Comment (pbrubeck) [inline symfem/elements/bernardi_raugel.py:133]: H1

File: symfem/elements/bernardi_raugel.py [hunk 2]
```diff
@@ -87,34 +92,22 @@ def __init__(self, reference: Reference, order: int):
...
                         )
                     )
 
             p = Lagrange(reference, 0, variant="equispaced")
-
             for i in range(3):
                 dofs.append(
                     DivergenceIntegralMoment(
-                        reference, x[i], p.dofs[0], entity=(3, 0), mapping="contravariant"
+                        reference, x[i], p.dofs[0], entity=(3, 0), mapping="identity"
```
Comment (pbrubeck) [inline symfem/elements/bernardi_raugel.py:107]: I am not sure why you have divergence moments, but the mapping for this dofs should contravariant. The canonical face DOFs are normal moments, and should map contravariantly.

**Review by pbrubeck — COMMENTED** [review]
I am leaving a review, even though this has been merged already

File: symfem/elements/bernardi_raugel.py (hunk 2 above)
Comment (mscroggs) [reply symfem/elements/bernardi_raugel.py:107]: I need to look at these again. I've opened #283

Comment (mscroggs) [reply symfem/elements/bernardi_raugel.py:107]: Looking at this element again. From the Bernardi-Raugel paper, it looks like these are right:

image

**Pull request merged**